cpp/bench/sg/benchmark.cuh

```diff
@@ -33,7 +34,7 @@ class Fixture : public MLCommon::Bench::Fixture {
  public:
   Fixture(const std::string& name)
     : MLCommon::Bench::Fixture(name,
-                               std::shared_ptr<deviceAllocator>(
+                               std::shared_ptr<MLCommon::deviceAllocator>(
```

Comment:
why not `raft::mr::device::allocator`? Is it to avoid more changes in some algorithms? 

Overall we have an issue in RAFT to track moving to use RMM directly, so we might wanna discuss something similar here https://github.com/rapidsai/raft/issues/97, what do you think? (Not for this PR of course)

Reply:
I think I was thinking that it would be more consistent with existing code, but I definitely prefer referring to the raft namespace directly. I'll tweak that right now.